.env.example

```
@@ -16,7 +16,7 @@ API_KEY=
 
 # Get your DSN from https://sentry.io
 SENTRY_DSN=https://your-key@o123456.ingest.sentry.io/7654321
-SENTRY_ENVIRONMENT=production
+SENTRY_ENVIRONMENT=development  # Default for development; change for staging/production
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)